packages/auth/src/grant/routes.ts

```diff
@@ -198,10 +277,14 @@ async function continueGrant(
   const continueToken = (ctx.headers['authorization'] as string)?.split(
     'GNAP '
   )[1]
-  const { interact_ref: interactRef } = ctx.request.body
 
-  if (!continueId || !continueToken || !interactRef) {
-    ctx.throw(401, { error: 'invalid_request' })
+  if (!continueId || !continueToken) {
+    ctx.throw(401, {
+      error: {
+        code: 'invalid_request',
```

Comment:
should this be `invalid_continuation`? 


> The AS MUST be able to tell from the client instance's request which specific ongoing request is being accessed, using a combination of the continuation URI and the continuation access token. If the AS cannot determine a single active grant request to map the continuation request to, the AS MUST return an invalid_continuation error ([Section 3.6](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#response-error)).


[Link](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-1.5)